<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/Logos/output-onlinepngtools (2).png</selectedon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Logo/Background.png);</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">#Logo{
background-color: transparent;
border-image: url(:/Logo/Logo.png);
background: none;
border: none;
background-repeat: none;
}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="LogoName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Brush Script MT</family>
      <pointsize>28</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#00009e;&quot;&gt;Anti Fraud Damage Detector&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HyperparametersButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#HyperparametersButton{
background-color: transparent;
border-image: url(:/Logo/Hyperparameters_up.png);
background: none;
border: none;
background-repeat: none;
}
#HyperparametersButton:pressed
{
   border-image: url(:/Logo/Hyperparameters_down.png);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeClassButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Classifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GraphButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#GraphButton{
background-color: transparent;
border-image: url(:/Logo/TrainGraph_up.png);
background: none;
border: none;
background-repeat: none;
}
#GraphButton:pressed
{
   border-image: url(:/Logo/TrainGraph_down.png);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>336</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#BackButton{
background-color: transparent;
border-image: url(:/Logo/Back_up.png);
background: none;
border: none;
background-repeat: none;
}
#BackButton:pressed
{
   border-image: url(:/Logo/Back_down.png);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>334</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Gui-pngs/Logos.qrc"/>
 </resources>
 <connections/>
</ui>
